package.json

```diff
@@ -113,6 +113,6 @@
     "src/**/*.scss": "stylelint"
   },
   "resolutions": {
-    "rollup": "4.20.0"
```

Comment:
wouldn't this cause build issues on s390x again? Or was the issue resolved in rollup 4.13.x?

Reply:
I think it is not needed anymore, as the lockfile still resolves to 4.20.0

Reply:
miss that, thanks for clarifying :+1: